blog/docs/articles/workflow-as-function-flow.md

```diff
@@ -0,0 +1,329 @@
+---
+title: Workflow as a Function Flow with Automatiko
+linkTitle: Knative Eventing backed Workflow as a Function Flow
+author: "[Maciej Swiderski](https://twitter.com/SwiderskiMaciek), Software Engineer @ OpenEnterprise."
+date: 2021-08-20
+description: Slicing business logic into functions and events.
+type: "blog"
+---
```

Comment:
@snneji is this no longer required for blogs? It looks like hugo frontmatter?

Reply:
Sorry for the late reply. I don't think this is required but I think it's useful information. In my opinion, the blog should show some of this information, however, it currently doesn't. I think we should keep it in case in the future we are able to show info like date, author, etc.